sdk/communication/communication-network-traversal/swagger/README.md

```diff
@@ -19,7 +19,7 @@ model-date-time-as-string: false
 optional-response-headers: true
 payload-flattening-threshold: 10
 use-extension:
```

Comment:
`package-name` should be `@azure/communication-network-traversal`
`package-version` should be `1.1.0-beta.1`

then please regenerate using autorest...the changes should be then apparent in `communication-network-traversal/src/generated/src/networkRelayRestClientContext.ts`

Reply:
Actually next version will be 1.1.0-beta.2 but yeah, I fixed these comments.